nss-day-cohort-41/tabloid-fullstack-bat-boy-apologists
# Bw tag delete

# Description
Users are now able to delete tags.
Fixes #15 
## Type of change
# Testing Instructions
Upon login, navigate to the Tag Management tab.
Choose the tag you would like to delete and confirm deletion.
# Checklist:
